services/backend/src/tmp/connectivity_pa_timeline_prot_conn.json

```diff
@@ -0,0 +1,26 @@
+{
+  "key": "ProtConn",
```

Comment:
Please use the same keys defined here: https://github.com/PEM-Humboldt/biotablero-backend/blob/develop/services/backend/src/tmp/connectivity_pa_current.json

Reply:
Keys prot and prot_conn were used.  Should I also change X and Y attributes?

Reply:
I don't see it necessary, how would you call them? aren't they called like this in human footprint?